tests/lib/types/types.test.ts

```diff
@@ -468,7 +468,7 @@ rule = {
         context.markVariableAsUsed("foo");
 
         context.report({ message: "foo", node: AST });
-        context.report({ message: "foo", loc: { line: 0, column: 0 } });
+        context.report({ message: "foo", loc: { start: {line: 0, column: 0}, end: { line: 1, column: 1 } } });
```

Comment:
This appeared to be an error in the current types, as `loc` should have `start` and `end` properties.